dojo/fixtures/test_type.json

```diff
@@ -719,5 +719,12 @@
         },
         "model": "dojo.test_type",
         "pk": 194
+    },
+    {
+        "fields": {
+            "name": "GitLab DAST Report"
+        },
+        "model": "dojo.test_type",
+        "pk": 196
     }
-]
+]
```

Comment:
This file is not needed anymore and will be removed soon. you must avoid changing it.

Reply:
Noted